app/views/newPaymentFlow.scala.html

```diff
@@ -1,17 +1,17 @@
 <header role="banner">
   <a class="glogo" href="#"><img src="#" alt="The Guardian"></a>
-  <div class="countries">
-    <span>United Kingdom (£)</span>
-    <ul>
-      <li><span>United Kingdom (£)</span></li>
-      <li><a href="#">United States (US$)</a></li>
-      <li><a href="#">Europe (€)</a></li>
-      <li><a href="#">New Zealand (NZ$)</a></li>
-      <li><a href="#">Canada (CA$)</a></li>
-      <li><a href="#">International (US$)</a></li>
-      <li><a href="#">Australia (AU$)</a></li>
+  <details class="countries">
+    <summary>United Kingdom (£)</summary>
```

Comment:
Couple of small questions, mainly because I'm not too familiar with `details`/`summary`! 🙂

Am I correct in thinking that `summary` is meant to be a summary of the content in the details box? So in this situation "United Kingdom (£)" is a summary of the list of country groups below?

What happens in the Microsoft family of browsers? Is the assumption that this will be controlled by JS anyway and so the functionality will work regardless of support for `details` in those browsers?

Reply:
> Am I correct in thinking that `summary` is meant to be a summary of the content in the details box?

It is a terrible name, but indeed you're right. I can add an `aria-label` on it `"Currently selected country: United Kingdom (currency: British Pounds)"` or something along those lines.

> What happens in the Microsoft family of browsers?

Yes, I'll add a polyfill if necessary.